{"level":"error","message":{"error":"Unexpected non-whitespace character after JSON at position 337 (line 17 column 1)","ip":"::1","method":"POST","stack":"SyntaxError: Unexpected non-whitespace character after JSON at position 337 (line 17 column 1)\n    at JSON.parse (<anonymous>)\n    at parse (/mnt/c/Users/AMOL/Desktop/Work/Backend/node_modules/body-parser/lib/types/json.js:77:19)\n    at /mnt/c/Users/AMOL/Desktop/Work/Backend/node_modules/body-parser/lib/read.js:123:18\n    at AsyncResource.runInAsyncScope (node:async_hooks:214:14)\n    at invokeCallback (/mnt/c/Users/AMOL/Desktop/Work/Backend/node_modules/raw-body/index.js:238:16)\n    at done (/mnt/c/Users/AMOL/Desktop/Work/Backend/node_modules/raw-body/index.js:227:7)\n    at IncomingMessage.onEnd (/mnt/c/Users/AMOL/Desktop/Work/Backend/node_modules/raw-body/index.js:287:7)\n    at IncomingMessage.emit (node:events:518:28)\n    at IncomingMessage.emit (node:domain:489:12)\n    at endReadableNT (node:internal/streams/readable:1698:12)","url":"/api/v1/parse"},"service":"backend-api","timestamp":"2025-07-29T12:29:27.766Z"}
{"level":"error","message":{"error":"Unexpected non-whitespace character after JSON at position 337 (line 17 column 1)","ip":"::1","method":"POST","stack":"SyntaxError: Unexpected non-whitespace character after JSON at position 337 (line 17 column 1)\n    at JSON.parse (<anonymous>)\n    at parse (/mnt/c/Users/AMOL/Desktop/Work/Backend/node_modules/body-parser/lib/types/json.js:77:19)\n    at /mnt/c/Users/AMOL/Desktop/Work/Backend/node_modules/body-parser/lib/read.js:123:18\n    at AsyncResource.runInAsyncScope (node:async_hooks:214:14)\n    at invokeCallback (/mnt/c/Users/AMOL/Desktop/Work/Backend/node_modules/raw-body/index.js:238:16)\n    at done (/mnt/c/Users/AMOL/Desktop/Work/Backend/node_modules/raw-body/index.js:227:7)\n    at IncomingMessage.onEnd (/mnt/c/Users/AMOL/Desktop/Work/Backend/node_modules/raw-body/index.js:287:7)\n    at IncomingMessage.emit (node:events:518:28)\n    at IncomingMessage.emit (node:domain:489:12)\n    at endReadableNT (node:internal/streams/readable:1698:12)","url":"/api/v1/parse"},"service":"backend-api","timestamp":"2025-07-29T12:29:31.826Z"}
{"address":"::","code":"EADDRINUSE","errno":-4091,"level":"error","message":"Uncaught Exception: listen EADDRINUSE: address already in use :::4000","port":4000,"service":"backend-api","stack":"Error: listen EADDRINUSE: address already in use :::4000\n    at Server.setupListenHandle [as _listen2] (node:net:1912:16)\n    at listenInCluster (node:net:1969:12)\n    at Server.listen (node:net:2074:7)\n    at startServer (C:\\Users\\AMOL\\Desktop\\Work\\Backend\\src\\server.ts:55:12)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)","syscall":"listen","timestamp":"2025-08-01T17:38:53.155Z"}
{"code":"ERR_SERVER_NOT_RUNNING","level":"error","message":"Error during server shutdown: Server is not running.","service":"backend-api","stack":"Error: Server is not running.\n    at Server.close (node:net:2323:12)\n    at Object.onceWrapper (node:events:638:28)\n    at Server.emit (node:events:524:28)\n    at Server.emit (node:domain:489:12)\n    at emitCloseNT (node:net:2383:8)\n    at processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-08-01T17:38:53.162Z"}
{"address":"::","code":"EADDRINUSE","errno":-4091,"level":"error","message":"Uncaught Exception: listen EADDRINUSE: address already in use :::4000","port":4000,"service":"backend-api","stack":"Error: listen EADDRINUSE: address already in use :::4000\n    at Server.setupListenHandle [as _listen2] (node:net:1912:16)\n    at listenInCluster (node:net:1969:12)\n    at Server.listen (node:net:2074:7)\n    at startServer (C:\\Users\\AMOL\\Desktop\\Work\\Backend\\src\\server.ts:55:12)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)","syscall":"listen","timestamp":"2025-08-01T17:39:16.643Z"}
{"code":"ERR_SERVER_NOT_RUNNING","level":"error","message":"Error during server shutdown: Server is not running.","service":"backend-api","stack":"Error: Server is not running.\n    at Server.close (node:net:2323:12)\n    at Object.onceWrapper (node:events:638:28)\n    at Server.emit (node:events:524:28)\n    at Server.emit (node:domain:489:12)\n    at emitCloseNT (node:net:2383:8)\n    at processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-08-01T17:39:16.649Z"}
{"address":"::","code":"EADDRINUSE","errno":-4091,"level":"error","message":"Uncaught Exception: listen EADDRINUSE: address already in use :::4000","port":4000,"service":"backend-api","stack":"Error: listen EADDRINUSE: address already in use :::4000\n    at Server.setupListenHandle [as _listen2] (node:net:1912:16)\n    at listenInCluster (node:net:1969:12)\n    at Server.listen (node:net:2074:7)\n    at startServer (C:\\Users\\AMOL\\Desktop\\Work\\Backend\\src\\server.ts:55:12)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)","syscall":"listen","timestamp":"2025-08-01T17:39:33.549Z"}
{"code":"ERR_SERVER_NOT_RUNNING","level":"error","message":"Error during server shutdown: Server is not running.","service":"backend-api","stack":"Error: Server is not running.\n    at Server.close (node:net:2323:12)\n    at Object.onceWrapper (node:events:638:28)\n    at Server.emit (node:events:524:28)\n    at Server.emit (node:domain:489:12)\n    at emitCloseNT (node:net:2383:8)\n    at processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-08-01T17:39:33.557Z"}
{"address":"::","code":"EADDRINUSE","errno":-4091,"level":"error","message":"Uncaught Exception: listen EADDRINUSE: address already in use :::4000","port":4000,"service":"backend-api","stack":"Error: listen EADDRINUSE: address already in use :::4000\n    at Server.setupListenHandle [as _listen2] (node:net:1912:16)\n    at listenInCluster (node:net:1969:12)\n    at Server.listen (node:net:2074:7)\n    at startServer (C:\\Users\\AMOL\\Desktop\\Work\\Backend\\src\\server.ts:55:12)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)","syscall":"listen","timestamp":"2025-08-01T17:40:15.668Z"}
{"code":"ERR_SERVER_NOT_RUNNING","level":"error","message":"Error during server shutdown: Server is not running.","service":"backend-api","stack":"Error: Server is not running.\n    at Server.close (node:net:2323:12)\n    at Object.onceWrapper (node:events:638:28)\n    at Server.emit (node:events:524:28)\n    at Server.emit (node:domain:489:12)\n    at emitCloseNT (node:net:2383:8)\n    at processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-08-01T17:40:15.674Z"}
{"address":"::","code":"EADDRINUSE","errno":-4091,"level":"error","message":"Uncaught Exception: listen EADDRINUSE: address already in use :::4000","port":4000,"service":"backend-api","stack":"Error: listen EADDRINUSE: address already in use :::4000\n    at Server.setupListenHandle [as _listen2] (node:net:1912:16)\n    at listenInCluster (node:net:1969:12)\n    at Server.listen (node:net:2074:7)\n    at startServer (C:\\Users\\AMOL\\Desktop\\Work\\Backend\\src\\server.ts:55:12)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)","syscall":"listen","timestamp":"2025-08-07T12:17:04.726Z"}
{"code":"ERR_SERVER_NOT_RUNNING","level":"error","message":"Error during server shutdown: Server is not running.","service":"backend-api","stack":"Error: Server is not running.\n    at Server.close (node:net:2323:12)\n    at Object.onceWrapper (node:events:638:28)\n    at Server.emit (node:events:524:28)\n    at Server.emit (node:domain:489:12)\n    at emitCloseNT (node:net:2383:8)\n    at processTicksAndRejections (node:internal/process/task_queues:89:21)","timestamp":"2025-08-07T12:17:04.734Z"}
